api/routes/project.js

```diff
@@ -1,8 +1,41 @@
+var express = require('express');
+var router = express.Router();
 var Project = require('../models/Projects');
 
-newProject = function() {
+// retrieves all projects
+router.get('/', function(req, res) {
+  Project.find({}, function(err, projects) {
+    if (err) {
+      res.sendStatus('Error in retrieving projects: ' + err)
```

Comment:
Function is not used properly, https://expressjs.com/en/api.html#res.sendStatus.